### PR TITLE
[IMP] l10n_ua: update national CoA and taxes to recent instruction

### PR DESCRIPTION
**Pull request by lem8r**
https://zakon.rada.gov.ua/laws/show/z0893-99

Add national account groups template

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ua/data/template/account.tax-ua_psbo.csv [hunk 1]
```diff
@@ -1,79 +1,79 @@
 "id","sequence","name","description","invoice_label","amount","type_tax_use","tax_group_id","price_include","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id"
-"sale_tax_template_vat20_psbo","9","Реалізація ПДВ 20%","","+ ПДВ 20%","20.0","sale","tax_group_vat20","","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_psbp_6431"
+"sale_tax_template_vat20_psbo","9","ПДВ 20%","","+ ПДВ 20%","20.0","sale","tax_group_vat20","","base","invoice",""
```
Comment (tsb-odoo) [inline addons/l10n_ua/data/template/account.tax-ua_psbo.csv:2]: Does this mean, the 6431 account should rather be used on the tax group as tax payable account? For the tax closing entry to work out the box in Ukraine?

File: addons/l10n_ua/data/template/account.tax-ua_psbo.csv [hunk 2]
```diff
@@ -1,79 +1,79 @@
...
 "","","","","","","","","","base","refund",""
 "","","","","","","","","","tax","refund",""
-"sale_tax_template_vat_not_psbo","14","Реалізація Не є об'єктом ПДВ","","Не є об'єктом ПДВ","0.0","sale","","","base","invoice",""
+"sale_tax_template_vat_not_psbo","14","Не є об'єктом ПДВ","","Не є об'єктом ПДВ","0.0","sale","","","base","invoice",""
 "","","","","","","","","","tax","invoice",""
 "","","","","","","","","","base","refund",""
 "","","","","","","","","","tax","refund",""
-"purchase_tax_template_vat20_psbo","19","Придбання ПДВ 20%","","+ ПДВ 20%","20.0","purchase","tax_group_vat20","","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_psbp_6441"
+"purchase_tax_template_vat20_psbo","19","ПДВ 20%","","+ ПДВ 20%","20.0","purchase","tax_group_vat20","","base","invoice",""
```
Comment (tsb-odoo) [inline addons/l10n_ua/data/template/account.tax-ua_psbo.csv:38]: Does this mean, the 6442 account should rather be used on the tax group as tax receivable account? For the tax closing entry to work out the box in Ukraine?

File: addons/l10n_ua/data/template/account.account-ua_psbo.csv [hunk 3]
```diff
@@ -208,14 +205,14 @@
...
-"ua_psbp_6431","6431","Податкові зобов’язання","liability_current","False",""
-"ua_psbp_6432","6432","Податкові зобов’язання непідтверджені","liability_current","False",""
-"ua_psbp_6433","6433","Коригування податкових зобов’язань","liability_current","False",""
-"ua_psbp_6434","6434","Податкові зобов’язання (роздріб)","liability_current","False",""
-"ua_psbp_6435","6435","Умовний продаж","liability_current","False",""
-"ua_psbp_6441","6441","Податковий кредит","liability_current","False",""
-"ua_psbp_6442","6442","Податковий кредит непідтверджений","liability_current","False",""
-"ua_psbp_6443","6443","Коригування податкового кредиту","liability_current","False",""
+"ua_psbp_6431","6431","Податкові зобов’язання","liability_current","True",""
+"ua_psbp_6432","6432","Податкові зобов’язання непідтверджені","liability_current","True",""
```
Comment (tsb-odoo) [inline addons/l10n_ua/data/template/account.account-ua_psbo.csv:209]: These accounts are used in tax repartition lines. 
It's quite incompatible to have bank reconciliation on tax provision accounts.
Please see "reconcile = true" as : visible in the bank reco widget when accounting is installed.

File: addons/l10n_ua/data/template/account.account-ua_psbo.csv [hunk 4]
```diff
@@ -208,14 +205,14 @@
...
 "ua_psbp_642","642","Розрахунки по обов’язкових платежах","liability_payable","True",""
-"ua_psbp_6431","6431","Податкові зобов’язання","liability_current","False",""
-"ua_psbp_6432","6432","Податкові зобов’язання непідтверджені","liability_current","False",""
-"ua_psbp_6433","6433","Коригування податкових зобов’язань","liability_current","False",""
-"ua_psbp_6434","6434","Податкові зобов’язання (роздріб)","liability_current","False",""
-"ua_psbp_6435","6435","Умовний продаж","liability_current","False",""
-"ua_psbp_6441","6441","Податковий кредит","liability_current","False",""
-"ua_psbp_6442","6442","Податковий кредит непідтверджений","liability_current","False",""
-"ua_psbp_6443","6443","Коригування податкового кредиту","liability_current","False",""
+"ua_psbp_6431","6431","Податкові зобов’язання","liability_current","True",""
```
Comment (tsb-odoo) [inline addons/l10n_ua/data/template/account.account-ua_psbo.csv:208]: If this account represents the debt towards the Tax administration, I would recommend Liability_Payable

File: addons/l10n_ua/data/template/account.account-ua_psbo.csv [hunk 5]
```diff
@@ -181,8 +178,8 @@
 "ua_psbp_521","521","Зобов’язання за облігаціями","liability_non_current","False",""
 "ua_psbp_522","522","Премія за випущеними облігаціями","liability_non_current","False",""
 "ua_psbp_523","523","Дисконт за випущеними облігаціями","liability_non_current","False",""
-"ua_psbp_531","531","Зобов’язання з фінансової оренди","liability_non_current","False",""
-"ua_psbp_532","532","Зобов’язання з оренди цілісних майнових комплексів","liability_non_current","False",""
+"ua_psbp_531","531","Зобов’язання з фінансової оренди","liability_non_current","True",""
```
Comment (tsb-odoo) [inline addons/l10n_ua/data/template/account.account-ua_psbo.csv:181]: here too, could liability_payable, non_trade be a better alternative?

File: addons/l10n_ua/data/template/account.account-ua_psbo.csv [hunk 6]
```diff
@@ -16,30 +16,29 @@
...
 "ua_psbp_143","143","Інвестиції непов’язаним сторонам","asset_fixed","False",""
-"ua_psbp_151","151","Капітальне будівництво","asset_fixed","False",""
-"ua_psbp_1521","1521","Придбання основних засобів","asset_fixed","False",""
-"ua_psbp_1522","1522","Виготовлення та модернізація основних засобів","asset_fixed","False",""
-"ua_psbp_1531","1531","Придбання  інших необоротних матеріальних активів","asset_fixed","False",""
-"ua_psbp_1532","1532","Виготовлення та модернізація інших необоротних матеріальних активів","asset_fixed","False",""
-"ua_psbp_1541","1541","Придбання нематеріальних активів","asset_fixed","False",""
-"ua_psbp_1542","1542","Виготовлення нематеріальних активів","asset_fixed","False",""
-"ua_psbp_155","155","Придбання (вирощування) довгострокових біологічних активів","asset_fixed","False",""
+"ua_psbp_151","151","Капітальне будівництво","asset_fixed","True",""
```
Comment (tsb-odoo) [inline addons/l10n_ua/data/template/account.account-ua_psbo.csv:34]: here too, could liability_payable, non_trade be a better alternative?

File: addons/l10n_ua/data/template/account.account-ua_psbo.csv [hunk 7]
```diff
@@ -16,30 +16,29 @@
...
 "ua_psbp_123","123","Права на комерційні позначення","asset_fixed","False",""
 "ua_psbp_124","124","Права на об’єкти промислової власності","asset_fixed","False",""
 "ua_psbp_125","125","Авторське право та суміжні з ним права","asset_fixed","False",""
 "ua_psbp_127","127","Інші нематеріальні активи","asset_fixed","False",""
-"ua_psbp_131","131","Знос основних засобів","expense_depreciation","False",""
-"ua_psbp_132","132","Знос інших необоротних матеріальних активів","expense_depreciation","False",""
-"ua_psbp_133","133","Накопичена амортизацiя нематеріальних активів","expense_depreciation","False",""
-"ua_psbp_134","134","Накопичена амортизація довгострокових біологічних активів","expense_depreciation","False",""
-"ua_psbp_135","135","Знос інвестиційної нерухомості","expense_depreciation","False",""
+"ua_psbp_131","131","Знос основних засобів","asset_fixed","False",""
```
Comment (tsb-odoo) [inline addons/l10n_ua/data/template/account.account-ua_psbo.csv:26]: LGTM

File: addons/l10n_ua/data/template/account.account-ua_psbo.csv [hunk 8]
```diff
@@ -111,8 +108,8 @@
 "ua_psbp_364","364","Розрахунки по гарантійному забезпеченню","asset_receivable","True",""
 "ua_psbp_365","365","Розрахунки з роздрібними покупцями","asset_receivable","True",""
 "ua_psbp_366","366","Розрахунки з вітчизняними покупцями (PoS)","asset_receivable","True",""
-"ua_psbp_3711","3711","Розрахунки за виданими авансами (в національній валюті)","asset_receivable","True",""
-"ua_psbp_3712","3712","Розрахунки за виданими авансами (в іноземній валюті)","asset_receivable","True",""
+"ua_psbp_3711","3711","Розрахунки за виданими авансами (в національній валюті)","asset_current","True",""
```
Comment (tsb-odoo) [inline addons/l10n_ua/data/template/account.account-ua_psbo.csv:111]: An alternative could be : asset_receivable + checkbox non trade set

File: addons/l10n_ua/data/template/account.tax-ua_psbo.csv [hunk 9]
```diff
@@ -1,79 +1,79 @@
...
 "","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund","ua_psbp_6441"
-"purchase_tax_template_vat0_psbo","22","Придбання ПДВ 0%","","ПДВ 0%","0.0","purchase","","","base","invoice",""
+"","","","","","","","","","tax","refund","ua_psbp_6442"
+"purchase_tax_template_vat0_psbo","22","ПДВ 0%","","ПДВ 0%","0.0","purchase","","","base","invoice",""
 "","","","","","","","","","tax","invoice",""
 "","","","","","","","","","base","refund",""
 "","","","","","","","","","tax","refund",""
-"purchase_tax_template_vat_free_psbo","23","Придбання звільнене від  ПДВ","","Звільнено від ПДВ","0.0","purchase","","","base","invoice",""
+"purchase_tax_template_vat_free_psbo","23","Звільнено від  ПДВ","","Звільнено від ПДВ","0.0","purchase","","","base","invoice",""
```
Comment (tsb-odoo) [inline addons/l10n_ua/data/template/account.tax-ua_psbo.csv:66]: We try to keep the Tax names as short as can be. 
Even if it means completing the tax description field to explain them if necessary.

Format = Rate% (Territory) (Scope) (Custom)
Territory (fiscal position) : 
Empty for domestic
EU for intra-community
EX for extracom/export/import
Scope
M for merchandise
G for goods
S for services
IG for investment goods
IS for investment services
Custom: if needed
Various specific abbreviation conventions:
Withholding => WH
Withholding income tax => WHI
Included/TTC => INC
Dividend distribution => DD
Triangular sale => Tri
Reverse charge => RC
Real Estate => RE

Examples
Sales Taxes :
21% => 21%
6% S. => 6% S
0% Cocont. => 0% Cocont

Purchase Taxes :
6% S. => 6% S
0% Cocont. S. => 0% S.Cocont
21% M. => 21% M
21% Non EU - Bien d'investissement => 21% EX IG
12% EU - Bien divers => 12% EU G
50% Non Déductible - Frais de voiture => 21% Car
10% Withholding tax royalties => 10% WH Royalties

**Review by tsb-odoo — COMMENTED** [review]
On top of this review, I suggest we develop the localization in English + provide the Ukrainian translation out of the box. It would be consistent with our standards and make the Ukrainian localization easier to understand / maintain in the future, without any impact on Ukrainian-speaking users.

**Comment by qdp-odoo** [comment]
superseded by https://github.com/odoo/odoo/pull/161351

**Pull request closed without merging**